### ISSUE TITLE
create app: xor-table

### ISSUE THREAD

**Issue by lcabrini** [mo-c/moses-cpp-learning#51]
the program should print a truth table for logical exclusive or

**Comment by mo-c** [comment]
Is this not the same as the #50

**Comment by lcabrini** [comment]
No, they are almost, but not exactly the same. OR (also called inclusive or) is slightly different from XOR (or exclusive or). The difference is that if the two operands A and B are both true then OR is true but XOR is false.

You use the | operator for OR, but the ^ operator for XOR.

**Comment by mo-c** [comment]
i have sent the code for the #50. I also made a comment to the difficulty i am facing with that exercise. Please look at it and guide me .
